### PR TITLE
project creates

### PR DESCRIPTION
**Pull request by AnvarjonShavqiyev**
React app is created

### CODE REVIEW

File: my-react-app/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Getting Started with Create React App
```
Comment (EugeneSinitsa) [inline my-react-app/README.md:1]: It is better to cleanup readme.md file. I prefer to remove default text

File: my-react-app/package.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
+{
+  "name": "my-react-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.17.0",
```
Comment (EugeneSinitsa) [inline my-react-app/package.json:6]: Why do we need version autoincrementing?

File: my-react-app/package.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
+{
+  "name": "my-react-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
```
Comment (EugeneSinitsa) [inline my-react-app/package.json:5]: split your deps between `dependencies` and `devDependencies`

File: my-react-app/public/index.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```
Comment (EugeneSinitsa) [inline my-react-app/public/index.html:1]: remove boilerplate

File: my-react-app/public/robots.txt [hunk 5]
```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
```
Comment (EugeneSinitsa) [inline my-react-app/public/robots.txt:1]: why do we need this file?

File: my-react-app/src/App.test.tsx [hunk 6]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
```
Comment (EugeneSinitsa) [inline my-react-app/src/App.test.tsx:5]: boilerplate. create another test

File: my-react-app/src/App.tsx [hunk 7]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
```
Comment (EugeneSinitsa) [inline my-react-app/src/App.tsx:7]: boilerplate

**Review by EugeneSinitsa — CHANGES_REQUESTED** [review]

**Review by EugeneSinitsa — APPROVED** [review]

**Review by VladKetris — APPROVED** [review]

**Comment by esnts** [comment]
You can merge this one

**Pull request merged**